src/components/ChatLog.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+
+
+
+const ChatLog = ({entries, heartClick}) => {
+    
+    const getChatLog = () => {
+        return entries.map((entry) => {
+            return (        
+                <ChatEntry 
+                key= {entry.id}
+                id= {entry.id}
+                sender= {entry.sender}
+                body= {entry.body}
+                timeStamp={entry.timeStamp}
+                liked={entry.liked}
+                heartClick= {heartClick}
+                />
+            );
+        });
+    };
+    return (
+        <div>
+            {getChatLog(entries)}
+        </div>
```

Comment:
Also, instead of writing a method `getChatLogs` to reference the return value from calling map() on entries, it's more common to directly iterate over the chat entries in your component like this without using a variable: 
```javascript
const ChatLog = ({entries, heartClick}) => {
    return (
     <div>
        {entries.map((entry) => (
            <ChatEntry 
                key={entry.id} 
                id={entry.id}
                sender={entry.sender}
                body={entry.body}
                timeStamp={entry.timeStamp}
                liked={entry.liked}
                heartClick={heartClick}
            />
        ))}
    </div>
    )
};